### PR TITLE
YOGA-15 Contact info form

### PR DESCRIPTION
**Pull request by angeliquerivera**
Will work on form submission issues on a later date, not a priority for now

### CODE REVIEW

File: app/components/ContactForm.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        required
+        className="w-full h-16 text-black  placeholder-gray-500 shadow-sm bg-transparent text-lg font-normal leading-7 rounded-md border  border-violet-600  focus:outline-none pl-4 mb-10"
+      />
+      <textarea
+        placeholder="Type your message here..."
+        value={message}
+        onChange={(e) => setMessage(e.target.value)}
+        required
+        className="w-full h-40 placeholder-gray-500 text-black shadow-sm bg-transparent text-lg font-normal leading-10 rounded-md border  border-violet-600  focus:outline-none pl-4 mb-16"
+      ></textarea>
```
Comment (chrismejia) [inline app/components/ContactForm.tsx:69]: Non-blocking nitpick

IIRC `<textarea>` can also be made a self-closing tag here, just like with `<input>`.

File: app/components/ContactForm.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        setResult(data.error || "An error occurred. Please try again.");
+      }
+    } catch (error) {
+      console.error("Submission error:", error);
+      setResult("An error occurred while submitting the form.");
+    }
+  };
+
+  return (
+    <form
```
Comment (chrismejia) [inline app/components/ContactForm.tsx:42]: Non-blocking

I'm thinking that I'll augment the form with validations and better state management using `react-hook-form`.
https://www.react-hook-form.com/get-started/

File: app/components/ContactInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+              className="text-blue-600"
+            >
+              allthingsselfcare@gmail.com
+            </a>
+          </span>
+        </div>
+        <div className="flex items-center mb-2">
+          <FontAwesomeIcon icon={faPhone} className="mr-2" />
+          <span>
+            <a href="tel:+8458212333" className="text-blue-600">
```
Comment (chrismejia) [inline app/components/ContactInfo.tsx:29]: Nice, you used the right tel href syntax 👍🏼

File: app/components/ContactInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+export default function ContactInfo() {
+  return (
+    <div className="w-full flex flex-col items-center rounded shadow-md">
+      <h2 className="text-2xl font-bold mb-4">Contact Info</h2>
+      <div className="flex flex-col items-start mb-4">
+        <div className="flex items-center mb-2">
+          <FontAwesomeIcon icon={faEnvelope} className="mr-2" />
+          <span>
+            <a
+              href="mailto:allthingsselfcare@gmail.com"
```
Comment (chrismejia) [inline app/components/ContactInfo.tsx:19]: Nice job with the email link.

File: app/contact/page.tsx [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
 export default function Contact() {
   return (
     <main>
-      <h2>Contact Us</h2>
+      <div className="min-h-screen flex items-center justify-center p-4">
+        <div className="flex flex-col md:flex-row space-x-4 w-full h-full max-w-5xl p-24">
+          <ContactInfo />
+          <ContactForm />
+        </div>
+      </div>
```
Comment (chrismejia) [inline app/contact/page.tsx:15]: Clean 💯

**Review by chrismejia — APPROVED** [review]
Looking forward to seeing this form in action.

File: app/components/ContactForm.tsx (hunk 1 above)
Comment (angeliquerivera) [reply app/components/ContactForm.tsx:69]: Edited and changed error text color to be more merciful to the eyeballs

File: app/components/ContactForm.tsx (hunk 2 above)
Comment (angeliquerivera) [reply app/components/ContactForm.tsx:42]: Dependency has been installed for future use/edits

**Review by chrismejia — APPROVED** [review]

**Pull request merged**